docs/sql-ref-functions.md

```diff
@@ -22,6 +22,17 @@ license: |
 Spark SQL provides two function features to meet a wide range of user needs: built-in functions and user-defined functions (UDFs).
 Built-in functions are commonly used routines that Spark SQL predefines and a complete list of the functions can be found in the [Built-in Functions](api/sql/) API document. UDFs allow users to define their own functions when the system’s built-in functions are not enough to perform the desired task.
 
+### Built-in Functions
+
+Spark SQL has some categories of frequently-used built-in functions for aggregtion, arrays/maps, date/timestamp, and JSON data.
+This subsection presents the usages and descriptions of these functions.
+
+ * [Aggregate Functions](sql-ref-functions-builtin.html#aggregate-functions)
```

Comment:
Super nit: order of bullets/words in the above statement is different from that in sql-ref-functions-builtin.html. .e. the alphabetical order in sql-ref-functions-builtin.html.

I am neutral on the order (current or consistent among them).

Reply:
Yea, actually, I sorted them by similar functionality, e.g., array and map. On the other hand, in `sql-ref-functions-builtin.html`, the script sorts them just by group name.... Yea, we can update the script for sorting them in the same order though, I just want to keep it simple.